docs/configuration/sinks/Opsgenie.rst

```diff
@@ -23,13 +25,30 @@ Configuring the OpsGenie sink
             name: ops_genie_sink
             api_key: OpsGenie integration API key  # configured from OpsGenie team integration
             teams:
-            - "noc"
-            - "sre"
+            - "noc"                  # Static team
+            - "$labels.team"         # Dynamic team routing based on alert label
```

Comment:
Please add also annotation to the docs. Maybe some customers using annotation and we are supporting it right?